.github/workflows/build.yml

```
@@ -5,7 +5,9 @@ on:
   pull_request:
     branches:
     - '*'
-  # repository_dispatch is a newer github-actions feature that will allow building from triggers other than code merge/PR
+  workflow_dispatch:
+    branches:
```

actionlint (1.7.12)

[error] expected "inputs" key for "workflow_dispatch" section but got "branches"

(syntax-check)